package.json

```diff
@@ -19,6 +19,10 @@
     "fix": "hallmark fix '**/*.md'",
     "postinstall": "husky install"
   },
+  "engines": {
+    "node": ">= 15.0.0",
```

Comment:
when I upgraded my local Node.js to 15.x I got this warning:

![grafik](https://user-images.githubusercontent.com/59574251/121523901-04d36500-c9f7-11eb-9187-13c670cdd5aa.png)

What's the reason for the update and why 15, not 16 then?

Reply:
15 is all that was required!
And when I locally checked which version I had via `nvm stable` I was on 15.
I now realize it could also be on 16, and that would be fine!
Happy to update it if you would like!